docs/how-to-guides/meson-args.rst

```diff
@@ -190,6 +190,10 @@ permanently in the project's ``pyproject.toml``:
    [tool.meson-python.args]
    setup = ['--vsenv']
 
+   # In case you want to build Windows wheels using cibuildwheel
+   [tool.cibuildwheel.windows]
+   config-settings = { setup-args = ["--vsenv"] }
```

Comment:
I think this won't properly work when you are building the wheel on cygwin/msys2

Reply:
That is true. But since the whole section was about MSVC, I saw little risk of someone adding this option while actually building the wheel on msys2. Do you think this needs further clarification, eg in the comment above the snippet?